assets/js/translator.js

```diff
@@ -1242,6 +1265,36 @@ function autoSelectDstLang() {
     $('#dstLangSelect').val(curDstLang).change();
 }
 
+// Check if the Language pair Exist or Not
+function checkLangPairAvailable(lang) {
+    if(languages[lang] !== undefined) {
+        var checkDstLang = pairs[iso639CodesInverse[lang]][0];
+        return checkDstLang !== undefined;
```

Comment:
Actually, I think this whole function can be `return languages[lang] && pairs[iso639CodesInverse[lang]].length`.

Reply:
Yeah thats short. :)

Reply:
Done :)

Reply:
and just return languages[lang] && pairs[iso639CodesInverse[lang]] is enough i think